src/components/split-pane/split-pane.scss

```diff
@@ -20,10 +20,6 @@ $split-pane-side-max-width: 28% !default;
   contain: strict;
 }
 
-[dir="rtl"] .split-pane {
-  flex-direction: row-reverse;
```

Comment:
because it is not the correct behavior.. I checked on an iPad, in the mail app, the menu part is on the right on rtl, and left on ltr